moped-editor/src/views/staff/StaffListView.js

```diff
@@ -110,12 +91,24 @@ const StaffListView = () => {
   }
 
   return (
-    <Page className={classes.root} title="Staff">
+    <Page title="Staff" sx={{ maxWidth: "100%" }}>
       {loading ? (
         <CircularProgress />
       ) : (
-        <Container className={classes.container}>
-          <Box className={classes.buttonContainer}>
+        <Box
+          sx={{
+            maxWidth: "100%",
+            minHeight: "100%",
+            padding: 3,
+          }}
+        >
```

Comment:
Thanks for reworking this! It looks identical to me. Out of curiousity, did you run into issue with the responsive behavior of the MUI Container component? I did in [this PR](https://github.com/cityofaustin/atd-moped/pull/1642/files#diff-24b2e90b33ce395f7da211c1e6cc5ff0fd81ed3ac0126c7869a82bfc5f37c21cR23-R24) and had to disable it which I guess was okay - might think about updating mine. I'm sure we'll find be many ways to match the styles!